### ISSUE TITLE
Move the new project template out of the cli package

### ISSUE THREAD

**Issue by javiertoledo** [boostercloud/booster#1078]
## Description

The `cli` package includes a full project template to be able to generate a scaffolding project with the `boost new:project` command.

This template, being embedded in the package itself is hard to maintain and cannot be separately improved. It'd be also desirable to allow Booster to use custom templates.

## Possible solution

Create a new "official" repository that contains the default template and use a tool like `[degit](https://github.com/Rich-Harris/degit)` to unpack it on demand.

**Comment by javiertoledo** [comment]
Given the advantages shown after migration to MS Rush, it would make sense to prepare multi-package rush templates for projects with frontend and backend subprojects

**Comment by NickSeagull** [comment]
Agree, although I'd prefer to use some strongly typed template engine like Dhall or maybe Projen
